### PR TITLE
AMP header initial work (and minor footer update)

### PR DESCRIPTION
**Pull request by nicl**
Adds a basic version of the header with the new design, some minimal body styling, and also minor changes to the footer for new colours.

To get this merged quicker, I've intentionally left stuff for further PRs:

* the AMP sidebar (veggie burger menu) styling
* some header and footer styling as the new versions aren't live yet and I don't want to overcommit on this work until they are/we know the design is fixed!

screen shot 2018-11-08 at 14 17 26

(...)

screen shot 2018-11-08 at 14 17 18

### CODE REVIEW

**Comment by nicl** [comment]
@paperboyo eyes like a hawk!

File: frontend/amp/components/Footer.tsx [hunk 1]
```diff
@@ -6,37 +6,35 @@ import InnerContainer from './InnerContainer';
 import { footerLinks, Link } from '../../lib/footer-links';
 
 const footer = css`
-    background-color: ${palette.neutral[20]};
+    background-color: #052962;
```
Comment (AWare) [inline frontend/amp/components/Footer.tsx]: We should be adding new colours to `palette.ts` and not be including raw colour values.

Comment (nicl) [reply frontend/amp/components/Footer.tsx]: Yeah, I agree. This is a new header/footer colour - I am going to talk to @zeftilldeath about how to add this.

Comment (nicl) [reply frontend/amp/components/Footer.tsx]: Okay this is now added under `brand.blue` although happy to hear alternative naming suggestions (@zeftilldeath guardian brand.)

**Comment by nicl** [comment]
@AWare what are your thoughts on merging/other changes? I've updated the palette-colour issues.

File: frontend/amp/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+import Logo from '@guardian/pasteup/logos/the-guardian.svg';
+import { screenReaderOnly } from '@guardian/pasteup/mixins';
+import { serif } from '@guardian/pasteup/fonts';
+import { pillarMap, pillarPalette } from '../../lib/pillars';
+import Sidebar from './Sidebar';
+import ArrowRight from '@guardian/pasteup/icons/arrow-right.svg';
+import { palette } from '@guardian/pasteup/palette';
+
+const headerStyles = css`
+    border-bottom: 1px solid #dcdcdc;
```
Comment (SiAdcock) [inline frontend/amp/components/Header.tsx]: Shall we add this to the palette (if it's not already there)?

Comment (SiAdcock) [reply frontend/amp/components/Header.tsx]: Note to self: would make a great lint rule!

**Pull request merged**